recipes/typhos/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "typhos" %}
+{% set version = "0.7.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/pcdshub/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: cdcc30a6e1a7eeb72f2a0b9db711dd28cbfc073e3c39e9ac2b7bfaa7fbb31821
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  entry_points:
+    - typhos = typhos.cli:main
+    - typhon = typhos.cli:main
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+
+  run:
+    - python >=3.6
+    - pyqt >=5
+    - qtpy
+    - coloredlogs
+    - numpy
+    - numpydoc
+    - ophyd
+    - pcdsutils
+    - pydm
+    - pyqtgraph
+    - qdarkstyle
+    - qtawesome
+    - qtconsole
+    - timechart
+
+test:
+  imports:
+    - typhos
+    - typhos.cli
+
```

Comment:
This is a stylish comment, but please remove the empty lines within a Section.

Reply:
Addressed in 9a96456a1f33c838472f0ce77b0829aba67f85f6 - I think I got them all.